code/modules/admin/DB ban/functions.dm

```diff
@@ -10,19 +10,19 @@ datum/admins/proc/DB_staffwarn_record(var/ckey, var/reason)
 	var/dbckey = sql_sanitize_text(ckey)
 
 	establish_db_connection()
-	if(!dbcon.IsConnected())
+	if(!SSdbcore.IsConnected())
 		to_chat(usr,"<span class='error'>Failed adding StaffWarn: db error</span>")
```

Comment:
While this file is being touched, some stylistic fixes:

```suggestion
		to_chat(usr,"<span class='error'>Failed adding staffwarn: Database connection error.</span>")
```

Reply:
I agree with the db replacement but StaffWarn seems nicer with camel casing since it's technically a compound word and a name.